markdown/usage/tutorials/step_by_step_institutional_profile.md

```diff
@@ -0,0 +1,617 @@
+-profile <your_cluster_name>,<you_profile_name>,test_tsv \
+--custom_config_base 'https://raw.githubusercontent.com/<your_github_user>/configs/<your_branch>'
+```
+
+To know if the profile is working you can check for things like the following:
+
+- Does the pipeline run summary displayed in your terminal at the beginning of the run display the right `config_profile_*` parameters?
+  - Note in some cases the _description_ may be of the test profile. However the URL should always be of your institutional profile
+- Does the pipeline run summary display the correct `containerEngine`?
+- Does your scheduler report jobs have been submitted to it from your pipeline?
+  - You can check this with `squeue` in SLURM or `qstat` in SGE, for example
+  - You should see job names beginning with `nf_` in your submission log
+
+> Tip: If you can't see the run summary header due to a higher number process status bars, you can run `export NXF_ANSI_LOG=false` before running the test command to use a more condense report without the fancy status information.
+
+## Make a PR into nf-core configs
+
+Once your testing works without any errors, it's time to make your config file official!
+
+Simply make a PR into the nf-core/configs repository, and request a review. Once approved and merged, any user can immediately use the institutional profile for all nf-core pipelines with just the `-profile` flag!
+
+```bash
+nextflow run nf-core/<pipeline> \
+-profile <your_cluster_name> \
+[...other params...]
+```
+
+Now time to sit back and feel good for helping yourself and all institutional Nextflow users make their experience of running pipelines as smooth and efficient as possible 😎.
+
+## Additional tips and tricks
+
+### If your cluster often have too-full harddisk space issues
+
+Some clusters often have very strict user HDD-space footprint restrictions (or just is often full). You can minimise the footprint that Nextflow runs use by using the `cleanup` directive.
+
+This directive goes _outside_ any of the config scopes, and is simply defined as:
+
+```nextflow
+cleanup = true
+```
+
+This directive will, on a _successful_ completion of a Nextflow run, automatically delete all intermediate files stored in the `work/` directory. Note that none of the 'published' files in the `--outdir` results directory will be deleted if the pipeline specifies to _copy_ results files, _however_ removal of these intermediate files will mean debugging 'silent fails' more difficult.
+
+To get around this, we suggest that you can optionally make an additional profile called debug:
+
+```nextflow
+debug {
+  cleanup = false
+}
+```
+
+which allows you to override the default `cleanup = true` behaviour, if you've set this as so.
+
+Alternatively, if your cluster utilises a scratch space to store intermediate files, this can be specified a the `process{}` scope of the profile with the [`scratch`](https://www.nextflow.io/docs/latest/process.html#scratch) directive. For example:
```

Comment:
```suggestion
Alternatively, if your cluster utilises a scratch space to store intermediate files, this can be specified a the `process` scope of the profile with the [`scratch`](https://www.nextflow.io/docs/latest/process.html#scratch) directive. For example:
```